doc/user/getting-help.rst

```diff
@@ -0,0 +1,44 @@
+.. vim: set filetype=rst
+
+===============
+How to get help
+===============
+
+**First**, be sure that you:
+
+#. Read the documentation (this site)
+
+#. Work through relevant examples in the `khmer-recipes repository <https://github.com/ged-lab/khmer-recipes>`__. Find **khmer-recipes** `documentation here <http://khmer-recipes.readthedocs.org/en/latest/#>`__.
```

Comment:
@mr-c: I'm not sure how to put line breaks without screwing up the restructured text formatting; can you advise?